### PR TITLE
Try testing the ACP module

### PR DESCRIPTION
**Pull request by VSEphpbb**

### CODE REVIEW

File: ad/manager.php [hunk 1]
```diff
@@ -39,7 +39,7 @@ public function __construct(\phpbb\db\driver\driver_interface $db, $ads_table, $
 	* Get specific ad
 	*
 	* @param	int		$ad_id	Advertisement ID
-	* @return	array	Advertisement data
+	* @return	mixed	Array with advertisement data, false if ad doesn't exist
```
Comment (senky) [inline ad/manager.php]: This probably doesn't belong here...

**Comment by senky** [comment]
You really want to achieve 99% code coverage, don't you? 😄

File: ad/manager.php (hunk 1 above)
Comment (VSEphpbb) [inline ad/manager.php]: Well, we are returning what comes from fetchrow, which is either gonna be an array or false.

Comment (VSEphpbb) [inline ad/manager.php]: Gotta fix it somewhere 😄

**Pull request merged**